WORKSPACE

```diff
@@ -112,6 +100,16 @@ http_archive(
     url = "https://github.com/abseil/abseil-cpp/archive/%s.tar.gz" % COM_GOOGLE_ABSL_COMMIT,
 )
 
+
+# re2 required for google tests
+http_archive(
```

Comment:
Huh, were tests even running before? Not sure how this wasn't part of the build earlier...

Reply:
They were. I think that [ZetaSQL was incidentally providing the headers previously](https://github.com/google/zetasql/blob/74b504c5933b5df3cab21055a5a36f788fcb9ffb/bazel/zetasql_deps_step_2.bzl#L395)